client/state/signup/steps/site-information/reducer.js

```diff
@@ -12,9 +12,8 @@ import { siteInformationSchema } from './schema';
 export default createReducer(
 	{},
 	{
-		[ SIGNUP_STEPS_SITE_INFORMATION_SET ]: ( state, { name, address, phone } ) => {
```

Comment:
Just to understand this better - so we're removing the name field here. We will be hiding the site title field in the About step though, if the site information step is enabled. So, for the site information step, we will be using two actions . `SIGNUP_STEPS_SITE_TITLE_SET ` and `SIGNUP_STEPS_SITE_INFORMATION_SET `?

Reply:
Ah, not sure about hiding the field. That's a good question. I assumed we'd pre-fill the form value with `siteTitle` to keep with the design.

Reply:
The business name field on the site information step essentially does the same job as the "What would you like to name your site?" field on the about step. I imagine we'll hide it on the about step because it still needs to show for other flows until we replace them all.

Reply:
I see that the checklist in the task mentions hiding the siteTitle field.
@fditrapani When the [site-type ](https://github.com/Automattic/wp-calypso/pull/28074)and this new site-information steps go live, the About step will show just one field, like below:
<img width="903" alt="screenshot 2018-10-31 at 7 01 20 pm" src="https://user-images.githubusercontent.com/1269602/47791317-6b704c80-dd3f-11e8-83ad-209494f6bc69.png">

Could this make the `About` step look a little empty, since we would have an entire step to just collect one field? I guess the subheader might need to change too, `Please answer these questions....` in the plural might have to go away?

Also, if the user selects site-type as `Business`, we will be hiding the phone and address fields. Same question as before - would hiding the field make this step look a little empty since we would have an entire step to just collect the one field `Business Name`?

Reply:
>If they pick any other option (blog, professional, ect..), they should see… Our about screen with the title, about field (vertical), and experience (if they're a new user)

@fditrapani: I was under the impression that we'd show the new steps for all segments - the only difference being that we'd hide phone, address, and email on the info step for non-business segments, and change any segment-specific wording (your business name to blogger name).

I think keeping the steps consistent and altering the content of theme based on segmentation/vertical will make future iterations easier. If you'd rather keep the About step, we can alter the tasks.

Reply:
aah ok. Yeah, that makes sense — thanks for clarifying. I think the best thing would be to push the change, then we can review the flow as a whole and make any updates after. How does that sound?